acceptance/helper.rb

```diff
@@ -228,7 +228,14 @@ def puppetdb_pids(host)
     pids
   end
 
-  def start_puppetdb(host, test_url="/pdb/meta/v1/version")
+  def start_puppetdb(host, legacy=false)
```

Comment:
What would you think about casing on the version instead of using `legacy`, i.e. `pdb_ver="3.x"`, or some convention like that?